src/style.css

```diff
@@ -1 +1,75 @@
 /* add styles here */
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: normal;
+  font-style: normal;
+}
+
+.form {
+  margin-top: 20px;
+}
+
+.big__input {
+  box-sizing: border-box;
+  width: 100%;
+  height: 70px;
+  padding-left: 62px;
+  outline: none;
+  background-image: url(./images/Search.svg);
+  background-repeat: no-repeat;
+  background-size: 19px;
+  background-position: 25px;
+  border: 1px solid rgba(225, 231, 237, 1);
+  box-shadow: 0 1 8 0 rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  font-family: Avenir, sans-serif;
+  font-size: 16px;
+  line-height: 21.86px;
+  color: rgba(61, 78, 97, 1);
+}
+
+.big__input:focus {
+  box-shadow: 0 4 4 0 rgba(0, 0, 0, 0.25);
+  background:
+    url(./images/Search.svg) no-repeat,
+    linear-gradient(180deg, #ffff 0%, #f6f6f7 100%);
+  background-size: 19px;
+  background-position: 25px;
+}
+
+.big__input:hover {
+  box-shadow: 0 3px 8px 0 rgba(61, 78, 97, 0.2);
+}
+
+.small__input {
+  box-sizing: border-box;
+  width: 100%;
+  height: 42px;
+  padding-left: 33px;
+  outline: none;
+  background-image: url(./images/Search.svg);
+  background-repeat: no-repeat;
+  background-position: 13px;
+  background-size: 11px;
+  border: 1px solid rgba(225, 231, 237, 1);
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.1);
+  font-family: Avenir, sans-serif;
```

Comment:
As with the previous font-family declaration, remember to include a generic fallback font. For example: `font-family: Avenir, Arial, sans-serif;`.